<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="5"
           failures="2"
           errors="0"
           skipped="0"
           time="411.649">
  <properties />
            <testcase name="testFBLogin" time="12.945" classname="com.houzify.scripts.NewUserRegTest">
                <failure type="java.lang.AssertionError"
                  message="null"
              >
<![CDATA[java.lang.AssertionError: null
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.fail(Assert.java:101)
  com.houzify.PO.HomePO.swipeHomeImg(HomePO.java:57)
  com.houzify.scripts.NewUserRegTest.testFBLogin(NewUserRegTest.java:154)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
  org.testng.SuiteRunner.run(SuiteRunner.java:254)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
      </failure>

          </testcase>
            <testcase name="testGoogleLogin" time="71.621" classname="com.houzify.scripts.NewUserRegTest">
                <failure type="java.lang.AssertionError"
                  message="null"
              >
<![CDATA[java.lang.AssertionError: null
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.fail(Assert.java:101)
  com.houzify.PO.NewUserRegPO.googleLogin(NewUserRegPO.java:382)
  com.houzify.scripts.NewUserRegTest.testGoogleLogin(NewUserRegTest.java:131)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
  org.testng.SuiteRunner.run(SuiteRunner.java:254)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="testSignUpwithGalleryImg" time="150.086" classname="com.houzify.scripts.NewUserRegTest" />
                <testcase name="testInvalidLogin" time="25.787" classname="com.houzify.scripts.NewUserRegTest" />
                <testcase name="testSignupWithCameraPic" time="151.210" classname="com.houzify.scripts.NewUserRegTest" />
      
</testsuite>
